### PR TITLE
Egress IP Doc updates for P/Z

### PR DESCRIPTION
**Pull request by alishaIBM**
Version(s) : 4.13+

Issue :
IBM P : https://issues.redhat.com/browse/MULTIARCH-3057
IBM Z : https://issues.redhat.com/browse/MULTIARCH-3058

Link to docs preview :
https://55979--docspreview.netlify.app/openshift-enterprise/latest/networking/openshift_sdn/assigning-egress-ips.html#nw-egress-ips-platform-support_egress-ips

QE review:
- [x] QE has approved this change.

Additional information:

cc : @SNiemann15

### CODE REVIEW

File: modules/nw-egress-ips-about.adoc [hunk 1]
```diff
@@ -48,6 +48,9 @@ Support for the egress IP address functionality on various platforms is summariz
 | Amazon Web Services (AWS) | Yes
 | Google Cloud Platform (GCP) | Yes
 | Microsoft Azure | Yes
+| IBM Z and LinuxONE | Yes
+| IBM Z and LinuxONE for {op-system-base-full} KVM | Yes
```
Comment (SNiemann15) [inline modules/nw-egress-ips-about.adoc:52]:
```suggestion
| {ibmzProductName} and LinuxONE for {op-system-base-full} KVM | Yes
```

File: modules/nw-egress-ips-about.adoc [hunk 2]
```diff
@@ -48,6 +48,9 @@ Support for the egress IP address functionality on various platforms is summariz
 | Amazon Web Services (AWS) | Yes
 | Google Cloud Platform (GCP) | Yes
 | Microsoft Azure | Yes
+| IBM Z and LinuxONE | Yes
```
[the remark this answers is not in the document]
Comment (alishaIBM) [reply modules/nw-egress-ips-about.adoc]: Ack.

**Comment by jboxman-rh** [comment]
@alishaIBM LGTM thanks

**Comment by jboxman-rh** [comment]
@alishaIBM, merged, thanks!

**Pull request merged**